actin/clinical/pull_transformed_nki_feed

```diff
@@ -2,32 +2,45 @@
 
 source message_functions || exit 1
 source locate_files || exit 1
+source actin_config || exit 1
 
+gcp_project=$(production_actin_nki_project)
 transformed_feed_directory=$(locate_actin_transformed_feed_directory)
 
 mkdir -p "${transformed_feed_directory}"
-transformed_contents=$(ls ${transformed_feed_directory})
+transformed_contents=$(ls "${transformed_feed_directory}")
 if [[ ${transformed_contents} ]]; then
 	echo -ne "Some previous transformed feed records exist in ${transformed_feed_directory}:\n${transformed_contents}\nDelete them? (y/N) "
 	read -r response
         if [[ "${response}" == "y" || "${response}" == "Y" ]]; then
 		rm -rf "${transformed_feed_directory}"
-		mkdir -p ${transformed_feed_directory}
+		mkdir -p "${transformed_feed_directory}"
 		info "Emptied ${transformed_feed_directory}."
 	fi
 else
 	info "No previous transformed feed found in ${transformed_feed_directory}"
 fi
 
 echo -e "\nTransformed feed records in cloud:"
-gs_path="actin-nki-clinical-feed-transformed"
+gs_path="${gcp_project}-clinical-feed-transformed"
 runs=$(gcloud storage ls gs://${gs_path} | grep "${gs_path}/2")
 
 echo "${runs}" | nl
-read -p "Which run would you like to pull (fill in the ID)? " choice
+read -rp "Which run would you like to pull (fill in the ID)? " choice
 selected_gs_path=$(echo "${runs}" | sed -n "${choice}p")
 
 if [[ ${selected_gs_path} ]]; then
-	gcloud storage cp -r "${selected_gs_path}" ${transformed_feed_directory}/
-	info "Copied ${selected_gs_path} to ${transformed_feed_directory}/"
+	path_without_trailing_slash="${selected_gs_path::-1}"
+	child_path="${path_without_trailing_slash##*/}"
+	mkdir -p "${transformed_feed_directory}/${child_path}"
+
+	file_list="$(gcloud storage ls "${selected_gs_path}")"
+	shared_data_bucket=$(locate_actin_shared_data_external_bucket "${gcp_project}")
+	for shared in $(gcloud storage ls "${shared_data_bucket}/record"); do
+		filename="${shared##*/}"
+		file_list=$(echo "${file_list}" | grep -v "${filename%%.*}")
+	done
+
+	gcloud storage cp ${file_list} "${transformed_feed_directory}/${child_path}/"
```

Comment:
Do you need a `$(cat ${file_list})` here? How does it know this is a list of files and not a URL?